<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Данные</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>761</width>
            <height>331</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2"/>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Функция потерь</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>761</width>
            <height>361</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3"/>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Точность</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>761</width>
            <height>361</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4"/>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Результат</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget_6">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>761</width>
            <height>361</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6"/>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>150</height>
           </size>
          </property>
          <property name="title">
           <string>Нейронная сеть</string>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_5">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>371</width>
             <height>131</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPushButton" name="file">
              <property name="text">
               <string>Выбрать файл</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Кол-во эпох:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="ln">
              <property name="text">
               <string>Обучить</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_8">
           <property name="geometry">
            <rect>
             <x>1</x>
             <y>10</y>
             <width>371</width>
             <height>131</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>100</height>
               </size>
              </property>
              <property name="text">
               <string>Ответ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plainTextEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>3467</width>
                <height>100</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
